docs/features/diff-viewer/diff-viewer-rest-api.md

```diff
@@ -0,0 +1,98 @@
+# Diff Viewer REST API
+
+The Diff Viewer REST API is intended for people who want to include visual regression testing in their build pipeline. E.g. you could make your build fail if the overall change rate is beyond a certain threshold.
+
+Parameters:
+
+* `branchName` / `buildName` in URL: build to compare with the comparison build
+* `comparisonName`: name under which the comparison will be available for selection
+* `banchName` / `buildName` in payload: comparison build
+
+## Calculate Configured Comparisons
+
+The comparisons configured in the `config.xml` file are automatically calculated after a build import (see [Publish Scenarioo Documentation Data](../../tutorial/Publish-Documentation-Data.md) and [Configuration of Automatic Comparisons](diff-viewer.md#configuration-of-automatic-comparisons)). You can use a calculation endpoint (see below) to poll for completion of these calculations.
+
+## Import Build and Calculate Comparison Synchronously
+
+This call triggers a build import and comparison calculation and also blocks until both are done. This operation can take a long time for larger builds. Consider using the non-blocking API calls instead. 
+
+`POST /rest/builds/{branchName}/{buildName}/comparisons/{comparisonName}/importAndCompare`
+
+Payload:
+
+`{ "branchName": "{comparisonBranchName}", "buildName": "{comparisonBuildName}" }`
+
+The response looks the same as the one returned by the [Get Calculation](#get-calculation) endpoint.
+
+In Scenarioo 3.0.0 the URL was different: `importBuild/{branchName}/{buildName}/{comparisonBranchName}/{comparisonBuildName}/{comparisonName}`.
```

Comment:
Please put that into the migration guide - in the docu we only document what is relevant for one version.

Reply:
I dont even think we had that at all in Scenarioo 3.0 since that was part of 4.0 to add this new service.

Reply:
Ok, guess we never had this endpoint in an official version, so I'll remove it everywhere.